python/cloudtik/providers/_private/aws/config.py

```diff
@@ -1261,6 +1298,14 @@ def _configure_network_resources(config, ec2, ec2_client,
         ec2_client.create_route(RouteTableId=private_route_table.id, DestinationCidrBlock='0.0.0.0/0',
                                 NatGatewayId=nat_gateway['NatGatewayId'])
 
+    # create VPC endpoint for S3
+    with cli_logger.group(
+            "Creating VPC endpoint",
+            _numbered=("[]", current_step, total_steps)):
+        current_step += 1
```

Comment:
This is a new step for creating workspace. we need
NUM_AWS_WORKSPACE_CREATION_STEPS = 7 -> NUM_AWS_WORKSPACE_CREATION_STEPS = 8